libs/shared/connected/src/lib/SignatureLists/useGetSignatureLists.ts

```diff
@@ -0,0 +1,44 @@
+import gql from 'graphql-tag'
+import { useQuery } from '@apollo/client'
+import { SignatureCollectionList } from '@island.is/api/schema'
+import { Query } from '@island.is/api/schema'
```

Comment:
Can we import from web/schema instead?

Reply:
I noticed that I didn’t even look at where the files were located that you were changing. What I’ve most often done when adding connected components for the web is put them under apps/web/components/connected that’s why I thought it was strange that you are relying on api schema types but since you are in libs/shared/connected you don’t have any other choice.